### ISSUE TITLE
Employment numbers jumping variably

### ISSUE THREAD

**Issue by backyardheaven** [Keriew/augustus#978]
So I noticed the Employment issue quite often with the unstable versions. The unemployment would jump all of a sudden by 200 to 300 workers, even when no apparent changes would happen in the city,
I managed to get 2 screenshots taken few minutes apart. 
<img width="251" alt="Screenshot 2023-10-01 181238" src="https://github.com/Keriew/augustus/assets/13154894/17d2372c-af63-4d64-829e-692fdebec658">
<img width="348" alt="Screenshot 2023-10-01 181201" src="https://github.com/Keriew/augustus/assets/13154894/0a5160c1-c6f1-4c79-bb69-f20b4e726e02">

**Comment by Keriew** [comment]
Sorry for taking a while before getting to this issue. Do you have the save with this problem that I could look at to examine what could be causing it?

**Comment by PaxBotanica** [comment]
@backyardheaven please forgive my ignorance if I'm wrong; Are you certain the population isn't just ageing? unemployment varies as births/death ratio varies according to the health provided; it causes fluctuations in % of working age in the populace. I have had similar unemployment throws which are really pronounced by the new higher game speed settings.

**Comment by crudelios** [comment]
Yes if you can mitigate the issue by using the setting "fixed 38% workforce" then that's the cause of the problem.

**Comment by Sephirex95** [comment]
This is indeed the unexpected consequence of aging pop.
Some clarity improvements in this regard are planned as this seems to be a common misconception :)
